compiler/optimizer/LocalAnticipatability.cpp

```diff
@@ -611,6 +612,11 @@ bool TR_LocalAnticipatability::updateAnticipatabilityForSupportedNodes(TR::Node
             if (!adjustInfoForAddressAdd(node, secondChild, seenDefinedSymbolReferences, seenStoredSymbolReferences, killedExpressions, storeNodes, block))
                return false;
             }
+         else if (node->isDataAddrPointer())
+	    {
+	    if (!adjustInfoForAddressAdd(node, node->getFirstChild(), seenDefinedSymbolReferences, seenStoredSymbolReferences, killedExpressions, storeNodes, block))
```

Comment:
currently, `adjustInfoForAddressAdd` for `dataAddrPointer` node always returns false. so, this looks like a placeholder we might do differently in `adjustInfoForAddressAdd` for `dataAddrPointer` node in the future.

Reply:
Thanks, I might try and fix this in this PR to not return false unconditionally for `dataAddrPointer` a bit later.

Reply:
On closer inspection, the only data address pointer load cases that won't get handled are `aladd/aiadd` or data address pointer nodes that also have `aladd/aiadd` or data address pointer load as children. The assumption was that these cases are rare in practice. So I believe data address pointer is not handled any worse than `aladd/aiadd` nodes were, originally. Agreed ?

Reply:
agreed